src/components/errorbars/index.js

```diff
@@ -70,13 +74,13 @@ errorBars.plot = function(gd, plotinfo, cd) {
             var trace = d[0].trace,
                 xObj = trace.error_x,
                 yObj = trace.error_y,
-                sparse = Plotly.Scatter.hasMarkers(trace) &&
+                sparse = subTypes.hasMarkers(trace) &&
                     trace.marker.maxdisplayed>0;
 
             if(!yObj.visible && !xObj.visible) return;
 
             d3.select(this).selectAll('g')
-                .data(Plotly.Lib.identity)
+                .data(Lib.identity)
               .enter().append('g')
```

Comment:
I'd vote for keeping the *classic* d3 style here.

Plus, it passed the eslint indent rule check, so nothing to worry.